analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityTask.java

```diff
@@ -0,0 +1,32 @@
+package ffb.analyzer.core.queue;
+
+
+
+public class Task implements Comparable<Task> {
+
+    public Long id;
```

Comment:
Is the intent that `PriorityQueueManager#getAvailableId` returns the same ID as the task with the highest ID? If so, should this just be an `int`?

Reply:
Updated to int.